### DISCUSSION TITLE
COMPRA CANCELAMENTO DE LICENÇA

### DISCUSSION THREAD

**Discussion by Ariana817185** [aclap-dev/video-downloadhelper#1552] [License & Purchase requests]
### What's your issue?

Olá! gostaria de cancelar e pedir um reembolso da minha compra de licença, pois não consegui baixar os vídeos que eu preciso

### The 10 first digits (only 10) of your license number or email address

arianadequeiroz.adv@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
